schemas/schemas/infoCourseInstance.json

```diff
@@ -93,6 +93,15 @@
                     }
                 }
             }
+        },
+        "groupBy": {
```

Comment:
Should we call this `groupAssessmentsBy` instead?

Reply:
Agreed.

Reply:
I made this change in [c7bf646](https://github.com/PrairieLearn/PrairieLearn/pull/4360/commits/c7bf646e4c21e794539d4e6b7c7487eac32029fb)